.github/workflows/tests.yml

```diff
@@ -145,7 +145,7 @@ jobs:
       - name: Install python dependencies
         run: |
           python -m pip install --disable-pip-version-check --user --upgrade pip
-          pip install --upgrade wheel setuptools
+          pip install --upgrade wheel "setuptools<65"
```

Comment:
That was my initial guess as well, but because it didn't work with only editing pyproject.toml, also added this. Now the reason it didn't work with just pyproject.toml is probably because I specified the wrong version ;)

Reply:
With tests now passing, it seems that the main fix was with `pyproject.toml`. Perhaps either remove or leave this line alone. The version of setuptools specified here probably does not make it to the editable install.

Reply:
Yep, just pushed a clean-up. We should probably also just remove the installation of wheel and setuptools here, as it should no longer be needed with pyproject.toml (but let's maybe leave such a clean-up for the main branch?)

Reply:
A fix to one or both branches is fine with me. (But probably both, so it no longer draws any attention). It's a harmless and mostly useless line that once served a purpose.